### ISSUE TITLE
How about re-producing the results of Doduo?

### ISSUE THREAD

**Issue by A11en0** [wbsg-uni-mannheim/wdc-sotab#1]
Thanks for your wonderful job.  I try to re-training a model on SOTAB dataset, but I cannot reach the performance that you report in the paper. Could you please share the code of re-produce of Doduo (or provide more implement details)? Thanks again!

**Comment by grxiao137** [comment]
Same problem here :-) Can you please release code for fine-tuning the Doduo model.

**Comment by kkorini** [comment]
Hello, thank you for checking our work!

I uploaded the code for fine-tuning Doduo in the folder baselines. The hyperparameters used for fine-tuning are batch size set to 32 and learning rate set to 5e-5.
